src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {

```

Comment:
<img width="194" alt="Screen Shot 2022-11-15 at 9 43 23 PM" src="https://user-images.githubusercontent.com/90692308/202011161-a859516e-0a3e-4b82-9827-6e86ba2f237d.png">
Service method get should return Driver not Optional